arch/arm/boot/dts/overlays/README

```diff
@@ -161,6 +161,13 @@ Params:
         and the other i2c baudrate parameters.
 
 
+Name:   adau1977-adc
+Info:   Overlay for activation of ADAU1977 ADC codec over I2C for conrtrol 
```

Comment:
There is a typo ("conrtrol" -> "control") and whitespace at the end of this line.